docs/content/tutorials/helm/index.md

```diff
@@ -0,0 +1,652 @@
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: webapp
+  namespace: {{ .Release.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: webapp
+  template:
+    metadata:
+      labels:
+        app: webapp
+    spec:
+      containers:
+      - name: webapp
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        env: 
+        - name: CONNECTION_REDIS_URL
+          valueFrom:
+            secretKeyRef:
+              name: redis-secret
+              key: url
+        ports:
+        - containerPort: 3000
+```
+
+The container that you will be working with is a ToDo application that uses Redis as a database. 
+
+- The container is configured to listen on port 3000.
+- The container will use the environment variable `CONNECTION_REDIS_URL` to connect to Redis.
+- This `CONNECTION_REDIS_URL` environment variable is populated by a Kubernetes Secret.
+
+You can deploy this application for the first time by following these steps:
+
+- Create the Kubernetes namespace `demo`
+- Create the Kubernetes Secret `redis-secret` containing the Redis URL.
+- Install the Helm chart.
+
+{{< alert title="💡 Redis" color="info" >}}
+For now you're not going to actually deploy Redis, the URL in this step is fake. You will deploy Redis using a Recipe later in the tutorial that will replace the fake URL contained within `redis-secret`.
+{{< /alert >}}
+
+Complete these steps by running the following commands:
+
+```bash
+kubectl create namespace demo
+kubectl create secret generic --namespace demo --from-literal=url=redis://fake-server redis-secret 
+helm upgrade demo ./Chart -n demo --install
+```
+
+The output should look similar to the following:
+
+```txt
+> kubectl create namespace demo
+namespace/demo created
+
+> kubectl create secret generic --namespace demo --from-literal=url=redis://redis-master:6379 redis-secret 
+secret/redis-secret created
+
+> helm upgrade demo ./Chart -n demo --install
+Release "demo" does not exist. Installing it now.
+NAME: demo
+LAST DEPLOYED: Wed Sep 13 01:05:19 2023
+NAMESPACE: demo
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+{{< alert title="⚠️ Chart Directory" color="warning" >}}
+If you see an error message like **Error: path "./Chart" not found** then you are in the wrong directory. Make sure your terminal is in the `./demo` directory of the `samples` repository.
+{{< /alert >}}
+
+Run the following command to check if everything is running:
+
+```bash
+kubectl get all -n demo
+```
+
+The output should look similar to the following:
+
+```txt
+> kubectl get all -n demo
+NAME                       READY   STATUS    RESTARTS   AGE
+pod/webapp-79d5dfb99-vhj9g   1/1     Running   0          2m48s
+
+NAME                   READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/webapp   1/1     1            1           2m49s
+
+NAME                             DESIRED   CURRENT   READY   AGE
+replicaset.apps/webapp-79d5dfb99   1         1         1       2m49s
```

Comment:
The spacing doesn't look right on this. Could you take a look and make sure the columns and data are all aligned?

Reply:
done